apps/wallet/src/ui/app/components/accounts/AccountItem.tsx

```diff
@@ -59,9 +60,13 @@ export const AccountItem = forwardRef<HTMLDivElement, AccountItemProps>(
 			>
 				{icon}
 				<div className="flex flex-col gap-1 overflow-hidden items-start">
-					<Text variant="pBody" weight="semibold" color="steel-darker" truncate>
-						{accountName}
-					</Text>
+					{!isActiveAccount ? (
```

Comment:
feel like this might be confusing UX

Reply:
also we might want to double check all the places AccountItem is used to make sure we don't need to disable editing altogether